src/manifest/MsQuicEtw.man

```diff
@@ -2134,6 +2134,20 @@
                 name="EcnCongestionCount"
                 />
           </template>
+          <template tid="tid_CONN_COUNT_COUNT">
+            <data
+                inType="win:Pointer"
+                name="Connection"
+                />
+            <data
+                inType="win:UInt32"
+                name="Count"
+                />
+            <data
+                inType="win:UInt32"
+                name="Count2"
+                />
```

Comment:
Are we intentionally being vague here?

Reply:
Not specifically, but I don't think it matters too much either way.